### ISSUE TITLE
[addtool] NucleiFuzzer

### ISSUE THREAD

**Issue by 0xKayala** [gwen001/offsectools_www#1506]
[tags]nucleifuzzer,nuclei,fuzzer,paramspider[/tags]

[short_descr]NucleiFuzzer is a Powerful Automation tool for detecting XSS, SQLi, SSRF, Open-Redirect, etc.. Vulnerabilities in Web Applications[/short_descr]

[link]https://github.com/0xKayala/NucleiFuzzer[/link]

[long_descr]NucleiFuzzer is an automation tool that combines ParamSpider and Nuclei to enhance web application security testing. It uses ParamSpider to identify potential entry points and Nuclei's templates to scan for vulnerabilities. NucleiFuzzer streamlines the process, making it easier for security professionals and web developers to detect and address security risks efficiently. Download NucleiFuzzer to protect your web applications from vulnerabilities and attacks.[/long_descr]

[image]https://github.com/0xKayala/NucleiFuzzer/assets/16838353/d3e6466c-8716-45bd-9bac-51f388ef8179[/image]
[image]https://github.com/0xKayala/NucleiFuzzer/assets/16838353/4bc2998b-b48d-4705-8ba7-16ff4c0aace7[/image]
[image]https://github.com/0xKayala/NucleiFuzzer/assets/16838353/bf911936-1eed-42bc-b81f-35d71a8ebf49[/image]

**Comment by gwen001** [comment]
Issue correctly handled, tool is waiting for human validation.

**Comment by 0xKayala** [comment]
Is there something wrong with the tool or is it pending for human validation?

**Comment by gwen001** [comment]
Tool has been refused by the team, feel free to get in touch if you have any question.

Thank you!

**Comment by 0xKayala** [comment]
May I know why the tool has been refused by your team so that I can work on it ?

**Comment by gwen001** [comment]
As said in the guidelines: "We definitely cannot add all bug hunters recon script unless they have a real plus value".

**Comment by 0xKayala** [comment]
Yes, I Agree with that and I have read the guidelines but here in this case my tool has a plus value which combines the functionality of two major open-source tools and gives the final output containing the security vulnerabilities. If you can't tag it under the tools section, can you tag it in any other section like scanner or fuzzer.
